packages/sockethub/src/platform-instance.ts

```diff
@@ -180,7 +184,7 @@ export default class PlatformInstance {
     const errorObject: ActivityObject = {
       context: this.name,
       '@type': 'error',
-      target: this.actor,
+      actor: this.actor,
```

Comment:
I'm wondering if target doesn't make more sense here, as the entity sending the error is not the user, but usually a server?

Reply:
Yarg! I don't know :D because in the case of a failed job, it's sending back the job object (actor is user) with the error message... additionally, in this case, it's sending the error object back as a message... so either that can both have the actor=user, or one can be actor=user and the other can be target=user ... does that make more sense? Or I could just remove this message altogether, though then hyperchannel wouldn't receive the failure since they aren't getting the failure message.

Reply:
Hyperchannel would then actually receive the failure message, because it's not being superceded by the message message which triggers the `offAny`.

> I don't know :D because in the case of a failed job, it's sending back the job object (actor is user) with the error message

Yeah, I'm also not sure, hence the question. The IRC connect failure sends a target with the user info, so at least it should be consistent between platforms then.

Reply:
The more I've been thinking of this the more a few things keep coming back to the forefront in Sockethubs expectations, eg. an actor property should always be  present. In this case, I think it makes sense that the actor is still the user@host because the report is that `user@host` generates the error. 

Additionally, to keep context and naming sensible, the PlatformInstance is what's assigning the `actor` based on its `actor` property, and to swap the actor to the target seems more confusing going forward. 

Finally, there's the idea of unsolicited/non response messages being the ones where the target is the user, and the actor is some other entity. I don't think necessarily it makes things more clear to swap these for error messages. 

So I'm thinking we keep things simple, stick with ensuring an actor if not present, and leave it at that (fixing the IRC one). What do you think?

Reply:
OK, let's do it like this then!